datasets/babi_qa/babi_qa.py

```diff
@@ -0,0 +1,2187 @@
+            type="shuffled-10k",
+            task_no="qa9",
+            version=VERSION,
+            description="This part of the config handles the `qa9` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa10",
+            version=VERSION,
+            description="This part of the config handles the `qa10` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa11",
+            version=VERSION,
+            description="This part of the config handles the `qa11` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa12",
+            version=VERSION,
+            description="This part of the config handles the `qa12` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa13",
+            version=VERSION,
+            description="This part of the config handles the `qa13` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa14",
+            version=VERSION,
+            description="This part of the config handles the `qa14` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa15",
+            version=VERSION,
+            description="This part of the config handles the `qa15` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa16",
+            version=VERSION,
+            description="This part of the config handles the `qa16` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa17",
+            version=VERSION,
+            description="This part of the config handles the `qa17` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa18",
+            version=VERSION,
+            description="This part of the config handles the `qa18` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa19",
+            version=VERSION,
+            description="This part of the config handles the `qa19` task of the bAbI `shuffled-10k` dataset",
+        ),
+        BabiQaConfig(
+            type="shuffled-10k",
+            task_no="qa20",
+            version=VERSION,
+            description="This part of the config handles the `qa20` task of the bAbI `shuffled-10k` dataset",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = (
+        "en-qa1"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+    )
```

Comment:
why have this one by default ?

Reply:
I am removing the default configurations in every file, hope that is cool.